### ISSUE TITLE
import error - Google AppEngine

### ISSUE THREAD

**Issue by andrewhillsc** [pallets/werkzeug#1210]
```
 (/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py:263)
Traceback (most recent call last):
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py", line 240, in Handle
    handler = _config_handle.add_wsgi_middleware(self._LoadHandler())
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py", line 299, in _LoadHandler
    handler, path, err = LoadObject(self._handler)
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py", line 85, in LoadObject
    obj = __import__(path[0])
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/devtools/cdbg/debuglets/python/imphook.py", line 114, in _ImportHook
    module = _real_import(name, globals, locals, fromlist, level)
  File "/base/data/home/apps/s~REDACTED/20171207t194833.406055908560331187/reportcard.py", line 9, in <module>
    from flask import Flask, jsonify, request
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/devtools/cdbg/debuglets/python/imphook.py", line 114, in _ImportHook
    module = _real_import(name, globals, locals, fromlist, level)
  File "/base/data/home/apps/s~REDACTED/20171207t194833.406055908560331187/lib/flask/__init__.py", line 17, in <module>
    from werkzeug.exceptions import abort
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/devtools/cdbg/debuglets/python/imphook.py", line 114, in _ImportHook
    module = _real_import(name, globals, locals, fromlist, level)
  File "/base/data/home/apps/s~REDACTED/20171207t194833.406055908560331187/lib/werkzeug/__init__.py", line 151, in <module>
    __import__('werkzeug.exceptions')
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/devtools/cdbg/debuglets/python/imphook.py", line 114, in _ImportHook
    module = _real_import(name, globals, locals, fromlist, level)
  File "/base/data/home/apps/s~REDACTED/20171207t194833.406055908560331187/lib/werkzeug/exceptions.py", line 71, in <module>
    from werkzeug.wrappers import Response
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/devtools/cdbg/debuglets/python/imphook.py", line 114, in _ImportHook
    module = _real_import(name, globals, locals, fromlist, level)
  File "/base/data/home/apps/s~REDACTED/20171207t194833.406055908560331187/lib/werkzeug/wrappers.py", line 37, in <module>
    from werkzeug.formparser import FormDataParser, default_stream_factory
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/devtools/cdbg/debuglets/python/imphook.py", line 114, in _ImportHook
    module = _real_import(name, globals, locals, fromlist, level)
  File "/base/data/home/apps/s~REDACTED/20171207t194833.406055908560331187/lib/werkzeug/formparser.py", line 14, in <module>
    from tempfile import SpooledTemporaryFile
ImportError: cannot import name SpooledTemporaryFile
```

**Comment by ThiefMaster** [comment]
ugh, goddamn google app engine... I guess we should fallback to a normal TemporaryFile if SpooledTemporaryFile is not available

**Comment by flebel** [comment]
`Flask` pins `werkzeug` to `>=0.7`. Pinning `werkzeug` to a `==0.12.x` version in my project fixed it.

**Comment by erlichmen** [comment]
I just released #1220 to handle this

**Comment by mitsuhiko** [comment]
This will be fixed by #1210.
